effectful/src/Effectful/Console/ByteString.hs

```diff
@@ -0,0 +1,49 @@
+-- | Lifted functions from "Data.ByteString" that are related to standard
+-- streams.
+-- Like the original module, you probably want to import this module qualified
+-- to avoid name clashes with the functions provided by "Prelude", e.g.:
+--
+-- > import Data.ByteString (ByteString)
+-- > import qualified Data.ByteString as BS
+-- > import Effectful.Console.ByteString (Console)
+-- > import qualified Effectful.Console.ByteString as Console
+--
+module Effectful.Console.ByteString
+  ( module Effectful.Console.Effect
```

Comment:
Can you export `Console` and `runConsole` explicitly (like e.g. `Effectful.FileSystem` does)? IMO it makes for a more readable haddock.

Reply:
Done.